xcube/core/dsio.py

```diff
@@ -549,3 +553,39 @@ def rimraf(path):
         except OSError:
             warnings.warn(f"failed to remove file {path}")
             pass
+
+
+def _get_path_or_store(path, client_kwargs, mode, root):
+    path_or_store = path
+    anon_mode = True
+    if not client_kwargs:
+        client_kwargs = {}
+
+    aws_credentials = ['aws_access_key_id', 'aws_secret_access_key']
```

Comment:
Why the `aws_` prefix in the keyword argument names? How can these be valid for OTC and GCP object storage API as well?

Reply:
It works for OTC as well, the provider is defined by the endpoint url. This is simply how the key is called to pass the credentials: https://s3fs.readthedocs.io/en/latest/#credentials

Reply:
But `aws` means Amazon Web Services. there this a wrong name for the key.  Please use a generic name such as `provider_acess_key_id` and `provider_access_key_secret`.